components/storage/blockdevice/COMPONENT_OSPIF/OSPIFBlockDevice.cpp

```diff
@@ -0,0 +1,1908 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "drivers/internal/SFDP.h"
+#include "platform/Callback.h"
+#include "OSPIFBlockDevice.h"
+#include <string.h>
+#include "rtos/ThisThread.h"
+
+#ifndef MBED_CONF_MBED_TRACE_ENABLE
+#define MBED_CONF_MBED_TRACE_ENABLE        0
+#endif
+
+#include "mbed_trace.h"
+#define TRACE_GROUP "OSPIF"
```

Comment:
Only 4 characters are allowed for TRACE_GROUP

Reply:
' #define TRACE_GROUP "OSPIF" " is copy from '#define TRACE_GROUP "QSPIF" ' in QSPIFBlockDevice.cpp .

Reply:
> ' #define TRACE_GROUP "OSPIF" " is copy from '#define TRACE_GROUP "QSPIF" ' in QSPIFBlockDevice.cpp .

That one should also be fixed but I'm happy if you just fix the new one.

Reply:
Could you tell me how i can fix it ?
Thanks!
@jeromecoutant @VeijoPesonen